Duplicati/Library/Backend/Jottacloud/Jottacloud.cs

```diff
@@ -146,6 +162,22 @@ public Jottacloud(string url, Dictionary<string, string> options)
             m_url_device = JFS_ROOT + "/" + m_userInfo.UserName + "/" + m_device;
             m_url        = m_url_device + "/" + m_mountPoint + "/" + m_path;
             m_url_upload = JFS_ROOT_UPLOAD + "/" + m_userInfo.UserName + "/" + m_device + "/" + m_mountPoint + "/" + m_path; // Different hostname, else identical to m_url.
+
+            m_threads = options.ContainsKey(JFS_THREADS) ? int.Parse(options[JFS_THREADS]) : 4;
+
+            if (!options.TryGetValue(JFS_CHUNKSIZE, out var tmp))
+            {
+                tmp = JFS_DEFAULT_CHUNKSIZE;
+            }
+
+            var chunksize = Utility.Sizeparser.ParseSize(tmp, "mb");
+
+            if (chunksize > int.MaxValue || chunksize < 1024)
+            {
+                throw new ArgumentOutOfRangeException(nameof(chunksize), string.Format("The chunk size cannot be less than {0}, nor larger than {1}", 1024, int.MaxValue));
```

Comment:
It might be helpful to include the units (B?) in the exception message.

Reply:
Agreed, but I tried to confirm to existing logic/ux (https://github.com/duplicati/duplicati/blob/master/Duplicati/Library/Main/Options.cs#L1343)

Reply:
Using FormatSizeString now resulting in: {"The chunk size cannot be less than 1.00 KB, nor larger than 2.00 GB\r\nParameter name: chunksize"}